docs/Deploying-Pega-on-openshift.md

```diff
@@ -194,12 +194,15 @@ Configure the following parameters so the backingservices.yaml Helm chart matche
 |elasticsearch: <ul><li>esConfig: elasticsearch.yml: xpack.security.enabled: "true”</li><li>esConfig: elasticsearch.yml: xpack.security.transport.ssl.enabled: "true”</li><li>volumeClaimTemplate: resources: requests: storage:  "\<30Gi>” </li></ul> | To run an internal ElasticSearch service (within your deployment), specify details for both the ElasticSearch security configuration and a cluster disk volume size. While the security pack should be enabled, the transport configuration should match whether an SSL certificate is available; by default it is enabled. For the disk volume size, the default is 30Gi; set this value to at least three times the size of your estimated search data size. | <ul><li>elasticsearch: esConfig: elasticsearch.yml: <ul><li>xpack.security.enabled: "true” </li><li>xpack.security.transport.ssl.enabled: "true” </li></ul></li><li>elasticsearch: volumeClaimTemplate: resources: requests: storage:  "\<30Gi>” </li></ul> |
 |Additional OpenShift-required parameters:<ul><li>securityContext:</li><li>podSecurityContext:</li><li>sysctlInitContainer:</li></ul> | Manually add ElasticSearch-recommended parameters to ensure that your SRS pods do not have a RunAsUser parameter and do not require an initialization container. For details, see [Examples/OpenShift](https://github.com/elastic/helm-charts/tree/master/elasticsearch/examples/openshift). | <ul><li>securityContext.runAsUser: null</li><li>podSecurityContext.fsGroup: null</li><li>    podSecurityContext.runAsUser: null </li><li>sysctlInitContainer.enabled: false</li></ul> |
 
 3. Save the file.
 
+4. To use an internal Elasticsearch cluster (srs.srsStorage.requireInternetAccess:true) for your deployment, you must download the Makefile file from the repository ( path from root : pega-helm-charts/charts/backingservices/Makefile) and replace <YOUR_NAMESPACE> with the namespace you used for the deployment, then run `$ make es-prerequisite`.
```

Comment:
very close...sorry I missed this...please use this formatting update>>
(path from root : pega-helm-charts/charts/backingservices/Makefile), replace <YOUR_NAMESPACE> with the namespace you used for the deployment, and then run